clar2wasm/src/tools.rs

```diff
@@ -341,8 +341,37 @@ pub fn interpret(snippet: &str) -> Result<Option<Value>, Error> {
     interpret_at(snippet, StacksEpochId::latest(), ClarityVersion::latest())
 }
 
+struct TestConfig;
+
+impl TestConfig {
+    /// Select a Clarity version based on enabled features.
+    pub fn clarity_version() -> ClarityVersion {
+        match () {
+            _ if cfg!(feature = "test-clarity-v1") => ClarityVersion::Clarity1,
+            _ if cfg!(feature = "test-clarity-v2") => ClarityVersion::Clarity2,
+            _ if cfg!(feature = "test-clarity-v3") || cfg!(feature = "test-clarity-latest") => {
+                ClarityVersion::Clarity3
+            }
+            _ => ClarityVersion::latest(),
+        }
+    }
+
+    /// Latest epoch for the enabled Clarity version.
+    pub fn latest_epoch_for_clarity_version() -> StacksEpochId {
```

Comment:
nit? unless we plan to have other epochs around in here (eg, some `earliest_epoch`)
```suggestion
    pub fn epoch() -> StacksEpochId {
```

Reply:
We do have other epochs. So, this is the reason of `latest` supported by a Clarity version.